docs/docs/guides/dashboard/on-premise/install.md

```diff
@@ -81,17 +82,17 @@ We facilitate authentication through [identity providers (IdP)](https://en.wikip
         
 We recommend authenticating using a [GitHub App](https://docs.github.com/en/apps/creating-github-apps/about-creating-github-apps/about-creating-github-apps) but you can also use the [OAuth App](https://docs.github.com/en/apps/oauth-apps/building-oauth-apps/creating-an-oauth-app). Make sure to include all essential environment variables specified by GitHub in the server environment. Absent variables will cause Tuist to overlook the GitHub authentication. To properly set up the GitHub app:
 - In the GitHub app's general settings:
-    - Copy the `Client ID` and set it as `TUIST_GITHUB_OAUTH_ID`
-    - Create and copy a new `client secret` and set it as `TUIST_GITHUB_OAUTH_SECRET`
```

Comment:
Isn't this a breaking change for on-premise organizations?

Reply:
No, we still use these old variables, they will just no longer be mentioned in the docs